SDK/AppCenterPush/Microsoft.AppCenter.Push.Android.Bindings/Microsoft.AppCenter.Push.Android.Bindings.csproj

```diff
@@ -24,7 +24,6 @@
     <WarningLevel>4</WarningLevel>
     <AndroidLinkMode>None</AndroidLinkMode>
     <AllowUnsafeBlocks>false</AllowUnsafeBlocks>
-    <AndroidClassParser>class-parse</AndroidClassParser>
```

Comment:
new visual studio for mac is updating csproj automatically, I checked UI the value does not change, seems like the new default value